adobecom/cc
# MWPW-140115 Release notes on slack channel when PR merged with Main branch

* Added workflow for send **release notes** to Slack channel(**cc-changelog**) when **PR** is **merged** with **main** branch
* This workflow will be triggered once the PR got merged in main branch

Resolves: [MWPW-140115](https://jira.corp.adobe.com/browse/MWPW-140115)

Note: No impact on cc pages

**Test URLs:**
- Before: https://main--cc--sivasadobe.hlx.live/creativecloud?martech=off
- After: https://mwpw-140115--cc--sivasadobe.hlx.live/creativecloud?martech=off

### How to Test

- Goto this PR: #326 
- Reopen this PR
- Change the title and description as needed, and then close the PR.
- Now the workflow triggered and you can see that on [Here](https://github.com/adobecom/cc/actions/workflows/merged-to-main.yaml) 
- Check the cc-changelog channel on Slack

Note: Because it is a testing workaround, it triggers the workflow with the Close action, but in actual flow, it is only called when the PR is merged (not closed).
